### ISSUE TITLE
Prose is breaking some styles

### ISSUE THREAD

**Issue by rain2o** [Myzwer/foothillschurch#40]
https://github.com/Myzwer/foothillschurch/blob/effc5bbf81497ae99f0561ab7849412fe9272bfd/form-success.php#L23-L24

I think `prose` might be messing with the styles of the content in some places. The form success is one example.

![Screenshot 2024-07-18 at 15 25 43](https://github.com/user-attachments/assets/b0a9f3e6-3c10-4d72-bc2d-b24ad711a08c)

**Comment by rain2o** [comment]
I found several more examples on https://foothillschurch.flywheelsites.com/other/ when testing each of the blocks. Long FAQ, Bio Block, Process Block, Announcement Block and Gallery Block are the ones I noticed.

**Comment by Myzwer** [comment]
So funny enough, this is a result of prose, but more a result of me trying to quick fix the issue, then going back and doing it correctly. Prose was putting way too much space between certain elements, so I created a class called "subtitle" that pulled the content up. Well that got old fast, so I went and modified the class. What you're seeing is remnants of that subtitle class over adjusting the content. This shouldn't be an issue moving forward.
